tests/core/engine_adapter/test_bigquery.py

```diff
@@ -745,10 +745,29 @@ def test_view_table_properties(make_mocked_engine_adapter: t.Callable, mocker: M
         },
     )
 
-    adapter.create_view("test_table", parse_one("SELECT 1"), table_properties={})
+    adapter.create_view(
+        "test_table",
+        parse_one("SELECT 1"),
+        table_description="some description",
+        view_properties={
+            "labels": exp.Array(
```

Comment:
can you use sqlglot builder here ?

Reply:
Sure, then I'll update [the existing code](https://github.com/TobikoData/sqlmesh/pull/2561/files/8fc49c3d99c6df4e564fe007d9898f8a5656a6c3#diff-31d4a537b0e60bd12859a67b8e1432a28bcf6f0fb48dc970111a65cdc3b2fb54R735-R744) too to be consistent